batch/batch/front_end/front_end.py

```diff
@@ -1320,58 +1408,12 @@ async def ui_get_billing(request, userdata):
     return await render_template('batch', request, userdata, 'billing.html', page_context)
 
 
-async def _query_billing_projects(db, user=None, billing_project=None):
-    args = []
-
-    where_conditions = ["billing_projects.`status` != 'deleted'"]
-
-    if user:
-        where_conditions.append("JSON_CONTAINS(users, JSON_QUOTE(%s))")
-        args.append(user)
-
-    if billing_project:
-        where_conditions.append('billing_projects.name = %s')
-        args.append(billing_project)
-
-    if where_conditions:
-        where_condition = f'WHERE {" AND ".join(where_conditions)}'
-    else:
-        where_condition = ''
-
-    sql = f'''
-SELECT billing_projects.name as billing_project,
-    billing_projects.`status` as `status`,
-    users
-FROM (
```

Comment:
The rebase for this was horrendous. Can you double check the query is still correct in utils.py?